contrib/dev/Vagrantfile

```diff
@@ -99,19 +111,25 @@ def populate_os_config(os, ip)
       address: '192.168.10.10',
     },
   ]
+  os.security_groups = [ 'skydive' ]
+end
+
+def box_url(type)
+  "https://download.fedoraproject.org/pub/fedora/linux/releases/27/CloudImages/x86_64/images/Fedora-Cloud-Base-Vagrant-27-1.6.x86_64.vagrant-#{type}.box"
 end
 
 Vagrant.configure(2) do |config|
   config.vm.define "dev" do |dev|
     dev.vm.hostname = "dev"
     dev.vm.synced_folder ".", "/vagrant", disabled: true
-    dev.vm.synced_folder SYNCED_FOLDER, "/home/vagrant/src/github.com/skydive-project/skydive", disabled: false, :type => "nfs", :nfs_udp => false
+    dev.vm.synced_folder SKYDIVE_HOST, SKYDIVE_GUEST, disabled: false, :type => "nfs", :nfs_udp => false
+    dev.vm.synced_folder ANSIBLE_HOST, ANSIBLE_GUEST, disabled: false, :type => "nfs", :nfs_udp => false
     dev.vm.box = "skydive/skydive-dev"
 
     if PREPARE_BOX then
       # Install Python for ansible
-      dev.vm.provision "common", type: "shell", inline: "sudo yum -y install python2 python-yaml libselinux-python"
-      dev.vm.provision "shared-folders", type: "shell", inline: "sudo chown vagrant:vagrant /home/vagrant/src /home/vagrant/src/github.com /home/vagrant/src/github.com/skydive-project"
+      dev.vm.provision "common", type: "shell", inline: "sudo yum -y install python2 python-yaml libselinux-python vim"
+      dev.vm.provision "shared-folders", type: "shell", inline: "sudo chown -R vagrant:vagrant #{GOPATH}/src"
```

Comment:
Why chowning the whole hierarchy ? On my setup, only the "skydive" directory was owned by "root". The sources comes from the host - using the shared folder - so you're changing the owner of the folders on the host

Reply:
At least of vboxfs we were getting "root" on the guest but the original (host) owner:group was different. How about setting `user: "vagrant", group: "vagrant"` in the `synced_folder` command?

Reply:
That must be a Windows problem because that should be the default. If setting `user` and `group` to `vagrant` fixes the problem, let's go for it.